### ISSUE TITLE
Pod package files not found

### ISSUE THREAD

**Issue by kroikie** [FirebaseExtended/flutterfire#863]
## Steps to Reproduce

First I downloaded the dev branch of flutter because cloud_firestore package was not working with the beta branch.
When I tried to run my app with the new flutter I got errors like:
```
fatal error: file <cloud_firestore/CloudFirestorePlugin.h> not found.
```
This was fixed by changing the target membership type from project to public:

![screen shot 2018-04-05 at 8 06 08 am](https://user-images.githubusercontent.com/26103979/38337350-45bb8de2-38a8-11e8-9c8b-0ec02c1e3d9a.png)

I'm guessing that this is a problem with the way flutter generates pod files.

## Logs

![screen shot 2018-04-05 at 8 11 30 am](https://user-images.githubusercontent.com/26103979/38337532-fd35b966-38a8-11e8-91b1-1d2b8d7cafad.png)

## Flutter Doctor
```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel dev, v0.2.8, on Mac OS X 10.13.3 17D102, locale en-AU)
[✓] Android toolchain - develop for Android devices (Android SDK 27.0.3)
[✓] iOS toolchain - develop for iOS devices (Xcode 9.2)
[✓] Android Studio (version 3.0)
[✓] Connected devices (2 available)

• No issues found!
```

**Comment by kroikie** [comment]
@code-rover

The issue at https://github.com/flutter/flutter/issues/16246 has been closed and moved here. Future collaboration on this issue will be done here.

**Comment by cvovan** [comment]
I had the same problem initially. However, I re-read the Setup section for the cloud_firestore plug-in at https://pub.dev/packages/cloud_firestore, and it said "Don't follow the steps named "Add Firebase SDK" and "Add initialization code" in the Firebase assistant".  
I recalled that I did updated the Podfile per Firebase assistant. So, I reverted the Podfile to the previous version, re-build, and I no longer have this issue. Good luck
